HW0/P6/P6B.py

```diff
@@ -16,7 +16,7 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(-6), 10**(-5), 10**(-4), 10**(-3), 10**(-2), 10**(-1), 1]
```

Comment:
consider using a function like numpy.logspace or a list comprehension here to make this cleaner